easybuild/easyblocks/f/fftw.py

```diff
@@ -203,8 +203,8 @@ def run_all_steps(self, *args, **kwargs):
                 comp_fam = self.toolchain.comp_family()
                 fftw_ver = LooseVersion(self.version)
                 if cpu_arch == POWER and comp_fam == TC_CONSTANT_GCC:
-                    # See https://github.com/FFTW/fftw3/issues/59 which applies to GCC 5.x - 10.x
-                    if prec == 'single' and fftw_ver <= LooseVersion('3.3.9'):
+                    # See https://github.com/FFTW/fftw3/issues/59 which applies to GCC 5 and above
+                    if prec == 'single':
```

Comment:
Rather than do this _forever_, how about
```suggestion
                    if prec == 'single' and fftw_ver <= LooseVersion('3.4'):
```
since this is likely valid for a very long time anyway